### PR TITLE
validate profile image and create fallback

### LINKED ISSUE #66: Profile Icon defaults to nonexistant file, and there is no fallback
Profile icon does not validate itself, so having no icon set or an invalid file will result in the dashboard CSS breaking.

### PR DESCRIPTION
**Pull request by matavach**
not 100% positive this is the most effect method of validation here. Let me know if it isn't.

fixes #66

### CODE REVIEW

File: modules/menus/dashboard/profile/index.ts [hunk 1]
```diff
@@ -24,7 +25,14 @@ const Profile = () => {
                     Widget.Icon({
                         hpack: "center",
                         class_name: "profile-picture",
-                        icon: image.bind("value"),
+                        icon: image.bind("value").as(i => {
+                            try{
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/profile/index.ts:29]:
```suggestion
                            try {
```

File: modules/menus/dashboard/profile/index.ts [hunk 2]
```diff
@@ -24,7 +25,14 @@ const Profile = () => {
...
+                                GdkPixbuf.Pixbuf.new_from_file(i)
+                                return i
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/profile/index.ts:31]:
```suggestion
                                return i;
```

File: modules/menus/dashboard/profile/index.ts [hunk 3]
```diff
@@ -24,7 +25,14 @@ const Profile = () => {
...
+                            } catch {
+                                return "avatar-default";
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/profile/index.ts:33]:
```suggestion
                                return "avatar-default-symbolic";
```
Lets use symbolic icons as they can be styled and colored.

File: options.ts [hunk 4]
```diff
@@ -652,7 +652,7 @@ const options = mkOptions(OPTIONS, {
                 logout: opt("pkill Hyprland"),
                 shutdown: opt("shutdown now"),
                 avatar: {
-                    image: opt("/home/jaskir/Pictures/Icons/900-900-max_catppuccin-mocha_hald8_GaussianRBF_lum1_shape96_near16.png"),
+                    image: opt("avatar-default"),
```
Comment (Jas-SinghFSU) [inline options.ts:655]:
```suggestion
                    image: opt("avatar-default-symbolic"),
```

**Comment by Jas-SinghFSU** [comment]
And once we've added the symbolic icons, we can style them in `dashboard.scss` with:
```css
    .profile-picture {
      color: $bar-buttons-icon;
      font-size: 7.5em;
    }
```

**Comment by matavach** [comment]
> And once we've added the symbolic icons, we can style them in `dashboard.scss` with:
> 
> ```css
>     .profile-picture {
>       color: $bar-buttons-icon;
>       font-size: 7.5em;
>     }
> ```

Very cool. I like it.

File: modules/menus/dashboard/profile/index.ts [hunk 5]
```diff
@@ -24,7 +25,14 @@ const Profile = () => {
                     Widget.Icon({
                         hpack: "center",
                         class_name: "profile-picture",
-                        icon: image.bind("value"),
+                        icon: image.bind("value").as(i => {
+                            try {
+                                GdkPixbuf.Pixbuf.new_from_file(i)
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/profile/index.ts:30]:
```suggestion
                                GdkPixbuf.Pixbuf.new_from_file(i);
```

**Pull request merged**